kalliope/core/RestAPI/views/neurons_view.py

```diff
@@ -0,0 +1,116 @@
+import glob
+import inspect
+import logging
+import os
+
+from flask import jsonify, Blueprint, request
+from kalliope.core.ResourcesManager import ResourcesManager, ResourcesManagerException
+from kalliope.core.Utils import Utils
+
+logging.basicConfig()
+logger = logging.getLogger("kalliope")
+
+LIST_EXCLUDED_DIR_NAME = ["__pycache__"]
+KALLIOPE_WEBSITE_NEURON_URL = "https://raw.githubusercontent.com/kalliope-project/kalliope-project.github.io/" \
+                              "sources/_data/community_neurons.yml"
+
+
+class NeuronsView(Blueprint):
+    def __init__(self, name, import_name, app=None, brain=None, settings=None):
+        self.brain = brain
+        self.settings = settings
+        self.app = app
+        super(NeuronsView, self).__init__(name, import_name)
+
+        # routes
+        self.add_url_rule('/neurons', view_func=self.get_neurons, methods=['GET'])
+        self.add_url_rule('/neurons/install', view_func=self.install_resource_by_git_url, methods=['POST'])
+
+    def get_neurons(self):
```

Comment:
this method could be defined as `methodclass`

Reply:
I'm not sure. I followed here the flask doc.
I think the method needs 'self'